lms/djangoapps/support/static/support/jsx/entitlements/data/api/client.js

```diff
@@ -0,0 +1,54 @@
+import 'whatwg-fetch'; // fetch polyfill
+import Cookies from 'js-cookie';
+
+import endpoints from './endpoints';
+
+export function requestEntitlements({usernameOrEmail}) {
```

Comment:
...maybe `userIdentifier` instead of `usernameOrEmail`? 

![alt-text](https://media.giphy.com/media/14tvbepZ8vhU40/giphy.gif)

#namingIsHard

Reply:
I'm going to leave it as usernameOrEmail, since thats explicit, userIdentifier makes me think of a uuid.